utils.c

```diff
@@ -18,10 +18,22 @@
 #include <dirent.h>
 #include <errno.h>
 #include <err.h>
+#include <unistd.h>
```

Comment:
Move this header to the preprocessor block around the OpenBSD version of `restrict_fsaccess()`.